zombienet/rococo-local-with-integritee-and-shell.toml

```diff
@@ -3,7 +3,7 @@ timeout = 10000
 
 [relaychain]
 chain = "rococo-local"
-default_command = "~/bin/polkadot-v0.9.43"
+default_command = "~/bin/polkadot-v0.9.42"
```

Comment:
Shouldn't we actually upgrade the relay chain binary to the newest one, to see if our code works with the current live version?

Reply:
The current setup still relies on locally built binaries. Somehow, my attempts to use --provider podman failed

Therefore, I think we need to adjust manually for now